### PR TITLE
[automation] Auto-update linters version, help and documentation

### PR DESCRIPTION
**Pull request by nvuillam**
[automation] Auto-update linters version, help and documentation

### CODE REVIEW

File: docs/descriptors/xml_xmllint.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To apply file formatting you must set `XML_XMLLINT_CLI_LINT_MODE: file` and `XML
...
 |-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
 | XML_XMLLINT_AUTOFORMAT                  | If set to `true`, it will reformat and reindent the output                                                                                                                                                          | `false`            |
-| XML_XMLLINT_INDENT                      | The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`                                                                                                                                            | `  `               |
+| XML_XMLLINT_INDENT                      | The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`                                                                                                                                            | ``                 |
```
Comment (echoix) [inline docs/descriptors/xml_xmllint.md:28]: This auto formatting seems problematic, but is not present after a force-push, maybe merged by another PR as is. It does not mean the same thing.

**Review by echoix — CHANGES_REQUESTED** [review]

File: docs/descriptors/xml_xmllint.md (hunk 1 above)
Comment (echoix) [reply docs/descriptors/xml_xmllint.md:28]: Is there other places where this happens?

Comment (nvuillam) [reply docs/descriptors/xml_xmllint.md]: It's the build doc & autofix that fight each other, happens for months , maybe years, it's not nice but not a big deal ^^

What is more wrong is that I forced the merge of a tiny PR and I didn't think that latest version of checkov would be broken :/

Comment (nvuillam) [reply docs/descriptors/xml_xmllint.md]: Posted a bug... :/ 

https://github.com/bridgecrewio/checkov/issues/5841

**Pull request merged**